project/make/.dockerversion

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+rm -rf autogenerated
+mkdir -p autogenerated/dockerversion
+cat > autogenerated/dockerversion/dockerversion.go<<EOF
+	// AUTOGENERATED FILE; see hack/make.sh
```

Comment:
This file location isn't accurate -- plugging in `$BASH_SOURCE` would be more accurate here.

```bash
cat > autogenerated/dockerversion/dockerversion.go <<EOF
// AUTOGENERATED FILE; see $BASH_SOURCE
package dockerversion
...
EOF
```